edge-hub/src/Microsoft.Azure.Devices.Edge.Hub.Service/modules/AuthModule.cs

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.Azure.Devices.Edge.Hub.Service.Modules
+{
+    using System.Threading.Tasks;
+
+    using Autofac;
+
+    using Microsoft.Azure.Devices.Edge.Hub.AuthAgent;
+    using Microsoft.Azure.Devices.Edge.Hub.Core;
+    using Microsoft.Azure.Devices.Edge.Hub.Core.Identity;
+    using Microsoft.Azure.Devices.Edge.Util;
+    using Microsoft.Extensions.Configuration;
+
+    class AuthModule : Module
+    {
+        private static readonly string defaultAddress = "http://localhost:7120/authenticate/";
```

Comment:
This should be config that can be passed in to both processes. 
Also, we should probably only include port and sub path (since it will only be available on localhost)

Reply:
it comes from config but has a default value. changed the config structure, so now it is just port/base url